src/tokenizer/definitions.py

```diff
@@ -475,6 +475,9 @@ def make_str(s):
     # Percentage and promille
     "%": ("%", 1.0),
     "‰": ("‰", 0.1),
+    # Velocity
+    "m/s": ("m/s", 1.0),
+    "km/klst.": ("m/s", 0.278),
```

Comment:
Maybe this should be `("m/s", 1000.0/(60*60))` to make the relationship clear (and exact)?

Reply:
Agree, done